Extend_volumes_Design.md

```diff
@@ -35,15 +35,13 @@ None
 ### REST API impact
 
 The following API will be added to support this capability.
-'''v1/{ProjectID}/volumes/{VolumeID}/action.
+'''v1/{ProjectID}/volumes/{VolumeID}/resize
```

Comment:
Should this be "extend" rather than "resize" because we only support changing it to a bigger size?

You have not made this change in the code yet, right?

Reply:
Looks like it has merged as "resize", but I agree that "extend" would be a better choice.